### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by burns238**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -64,7 +64,7 @@ play.modules.enabled += "config.DIModule"
 # ~~~~~
 # The secret key is used to secure cryptographics functions.
 # If you deploy your application to several instances be sure to use the same key!
-play.crypto.secret = "OZgF7ykybYlo2lyfj11A5IWLACBsr5Ar5n0Ts4JUdN4GPUdUPgBQb4u1zjy2ItcE"
```
Comment (versatil56) [inline conf/application.conf:67]: do we fully understand the implications of this change?

File: conf/application.conf [hunk 2]
```diff
@@ -87,19 +87,6 @@ play.i18n.langs = ["en"]
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
 play.http.router = prod.Routes
 
-# Logger
-# ~~~~~
-# You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
```
Comment (versatil56) [inline conf/application.conf:92]: why is all this gone?

File: conf/application.conf [hunk 3]
```diff
@@ -110,7 +97,7 @@ api {
   gateway.context = "organisations/interest-restriction"
 }
 
-httpHeadersWhitelist = [ "Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id" ]
```
Comment (versatil56) [inline conf/application.conf:113]: any implicatations with this?

**Review by versatil56 — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 3 above)
Comment (burns238) [reply conf/application.conf:113]: From the READMEs of the bootstrap libraries both this and the change to play.crypto.secret look to be simply renaming the settings.

File: conf/application.conf (hunk 2 above)
Comment (burns238) [reply conf/application.conf:92]: The compiler was giving warnings saying that any logging config in the .conf will be ignored and we should use a logback file, which we already are.

**Review by versatil56 — APPROVED** [review]

**Pull request merged**